Ecosystem Partners/getting-started-with-jboss-blueprint.md

```diff
@@ -0,0 +1,153 @@
+{{{
+  "title": "Getting Started with JBoss - Blueprint",
+  "date": "05-05-2015",
+  "author": "<a href='https://twitter.com/KeithResar'>@KeithResar</a>",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+
+
+### Description
+
+<img src="//d33np9n32j53g7.cloudfront.net/assets/stacks/jboss/img/jboss-stack-110x117-12361d85ff78442ab801caef8baf4d63.png" style="border:0;float:right;max-width:250px">
```

Comment:
What do you want it to contain?

From: Matthew Osborn <notifications@github.com<mailto:notifications@github.com>>
Reply-To: CenturyLinkCloud/PublicKB <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Tuesday, May 5, 2015 at 12:40 PM
To: CenturyLinkCloud/PublicKB <PublicKB@noreply.github.com<mailto:PublicKB@noreply.github.com>>
Cc: Keith Resar <keith.resar@ctl.io<mailto:keith.resar@ctl.io>>
Subject: Re: [PublicKB] Ecosystem/Update Bitnami KB Stacks: wildfly, alfresco, tomcat, dokuwiki, drupal, prestashop, abantecart, django, jboss, diaspora, joomla, owncloud, elasticsearch, phpbb, jenkins, solr, mean (#115)


In Ecosystem Partners/getting-started-with-jboss-blueprint.md<https://github.com/CenturyLinkCloud/PublicKB/pull/115#discussion_r29693019>:

> @@ -0,0 +1,153 @@
> +{{{
> +  "title": "Getting Started with JBoss - Blueprint",
> +  "date": "05-05-2015",
> +  "author": "<a href='https://twitter.com/KeithResar'>@KeithResar</a>",
> +  "attachments": [],
> +  "contentIsHTML": false
> +}}}
> +
> +
> +
> +### Description
> +
> +<img src="//d33np9n32j53g7.cloudfront.net/assets/stacks/jboss/img/jboss-stack-110x117-12361d85ff78442ab801caef8baf4d63.png" style="border:0;float:right;max-width:250px">


Can we also generate alt attributes for images?

—
Reply to this email directly or view it on GitHub<https://github.com/CenturyLinkCloud/PublicKB/pull/115/files#r29693019>.